### DISCUSSION TITLE
Custom MTU in podman deployment

### DISCUSSION THREAD

**Discussion by daniel-duchon** [uyuni-project/uyuni#9356] [Q&A]
Hi,

can anyone tell me how I can adjust the MTU size of the network bridge to 1280 for a podman deployment? Admittedly, I come from the docker and docker compose corner. Podman and mgrctl as well as mgradm is still new territory for me.

Greetings

**Comment by cbosdo** [comment]
At the moment `mgradm` doesn't offer any option to adjust the mtu of the podman network it creates. According to [podman network create's man page](https://docs.podman.io/en/latest/markdown/podman-network-create.1.html#opt-o-option) it could use an option to set it.

In case you want to implement this enhancement, the code generating the podman network is [here](https://github.com/uyuni-project/uyuni-tools/blob/main/shared/podman/network.go#L30). I would happily review a PR or guide you through creating it.

**Reply by daniel-duchon** [reply]
Unfortunately, I have no experience at all with go.

I'll ask my network if anyone is interested in helping out.

**Reply by cbosdo** [reply]
OK. Keep us posted if you can find someone willing to help. Otherwise you could also find an enhancement issue for uyuni-tools.
